### PR TITLE
Add support for the muliple widgets for restapi

### PR DESCRIPTION
**Pull request by rwx788**
See https://github.com/libyui/libyui-rest-api/pull/3

### CODE REVIEW

File: package/libyui-qt.changes [hunk 1]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Nov 29 14:42:52 UTC 2019 - Rodion Iafarov <riafarov@suse.com>
+
+- Add support to operate on many widgets with rest-api
+- 2.52.0
```
Comment (shundhammer) [inline package/libyui-qt.changes]: Please also mention the so_version 11 bump.

File: src/YQRichText.h [hunk 2]
```diff
@@ -151,6 +151,12 @@ class YQRichText : public QFrame, public YRichText
      **/
     virtual void setHScrollValue( const std::string & newValue ) override;
 
+    /**
+     * Derived classes should implement this, method is used to trigger event
+     * like user has clicked link in the RichText
+     **/
+    virtual void clickLink( const std::string & url );
```
Comment (shundhammer) [inline src/YQRichText.h]: This should become `activateLink()` in libyui, so this will need to be renamed here as well.

**Comment by shundhammer** [comment]
Besides the two minor things, LGTM.

File: package/libyui-qt.changes (hunk 1 above)
Comment (rwx788) [reply package/libyui-qt.changes]: Done, thanks.

File: src/YQRichText.h (hunk 2 above)
Comment (rwx788) [reply src/YQRichText.h]: renamed as in libyui.

**Pull request merged**